app/use_cases/accounts/apps/evolution_apis/message/delivery_job.rb

```diff
@@ -1,26 +1,24 @@
 class Accounts::Apps::EvolutionApis::Message::DeliveryJob < ApplicationJob
-
   self.queue_adapter = :good_job
   def perform(event_id)
     @event = Event.find(event_id)
     if should_delivery?(@event)
       phone_id = group? ? @event.contact.additional_attributes['group_id'] : @event.contact.phone
 
-      result = Accounts::Apps::EvolutionApis::Message::Send.call(
-        @event.app,
-        phone_id,
-        @event.content
-      )
+      result = Accounts::Apps::EvolutionApis::Message::Send.new(@event.app,
+                                                                phone_id,
+                                                                @event).call
```

Comment:
Aqui talvez passar direto o evento fica melhor ex:

```suggestion
      result = Accounts::Apps::EvolutionApis::Message::Send.new(@event).call
```

Reply:
Consegui remover o @event.app, mas o `phone_id` eu n vi outro jeito de pegar ele a n ser por parametro.
Ficou assim: 
```ruby
result = Accounts::Apps::EvolutionApis::Message::Send.new(@event).call
```